app/views/shared/_back_to_login.html.erb

```diff
@@ -0,0 +1,3 @@
+<div data-go-back-link='true' style='display:none;line-height:3'>
```

Comment:
Is there a particular css file that this should go in? Is it possible to style on the `data-go-back-link=‘true’` or should I add an id for this?


> On May 1, 2015, at 12:40 PM, Jessie Keck <notifications@github.com> wrote:
> 
> In app/views/shared/_back_to_login.html.erb <https://github.com/sul-dlss/sul-requests/pull/107#discussion_r29527795>:
> 
> > @@ -0,0 +1,3 @@
> > +<div data-go-back-link='true' style='display:none;line-height:3'>
> I think the display:none is okay in a style tag when we want to be absolutely sure it's hidden immediately even before any css is loaded, but we should avoid styling things like line-height directly in the markup like this.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/sul-dlss/sul-requests/pull/107/files#r29527795>.
>